src/Editor/Resources_BACKUP_7056.cpp

```diff
@@ -0,0 +1,261 @@
+#include "Resources.hpp"
+
+#include <Engine/Texture/TextureAsset.hpp>
+#include <Engine/Animation/AnimationClip.hpp>
+#include <Engine/Animation/AnimationController.hpp>
+#include <Engine/Animation/Skeleton.hpp>
+#include <Engine/Geometry/Model.hpp>
+#include <Engine/Audio/SoundBuffer.hpp>
+#include <Engine/Script/ScriptFile.hpp>
+#include <Engine/Hymn.hpp>
+#include <Engine/Manager/Managers.hpp>
+#include <Engine/Manager/ResourceManager.hpp>
+#include <Engine/Util/FileSystem.hpp>
+
+using namespace std;
+
+string ResourceList::Resource::GetName() const {
+    switch (type) {
+    case Type::SCENE:
+<<<<<<< HEAD
+        return scene;
+    case Type::ANIMATION_CLIP:
+        return animationClip->name;
+    case Type::ANIMATION_CONTROLLER:
+        return animationController->name;
+    case Type::SKELETON:
+        return skeleton->name;
+=======
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Statements following return, break, continue, goto or throw will never be executed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreachableCode)